backend/models/eventTemplate.model.js

```diff
@@ -1,20 +1,27 @@
 const mongoose = require('mongoose');
-
+const recurIntervalEnums = ['daily','weekly','bi-weekly','monthly'];  // daily, weekly, bi-weekly, monthly 
+const monthWeekEnums = [1,2,3,4,5];                                   // 1st week, 2nd week, 3rd week, 4th week, 5th week - a month can have either 4 weeks or 5 weeks
+const weekDayEnums = [0, 1, 2, 3, 4, 5, 6];                           // sunday,monday,tuesday, wednesday,thrursday,fridaya,saturdaya
 mongoose.Promise = global.Promise;
 
 const eventTemplateSchema = mongoose.Schema({
     name: { type: String },
+    belongsToProjectID:{type:Number}, // Ative project an event is a child of, if any (structured)
+    eventManagerID:{type:Number},     // ID of user that is responsible for managing event
+    locationZone: { type: mongoose.Schema.Types.ObjectId,  ref: 'TimeZone'  },                                // HfLA locations project participates at (predefined in dictionary)
+    locationName:{ type: String },    // Name of event location
```

Comment:
This is my super-nitpickyest comment of all, but the number of spaces after the ":" on your code isn't consistent -- sometimes it's a single space, sometimes it's double.  It makes my eye keep going back to it because inconsistencies make my eye think there's a problem when I'm scanning code.  Obviously, not an functional problem, but it would make me smile if they were consistent.

Reply:
ref: should possibly be LocationZone and NOT timeZone